browsergym/experiments/src/browsergym/experiments/agent.py

```diff
@@ -27,8 +27,8 @@ def default_obs_preprocessor(obs: dict) -> dict:
 class AgentInfo:
     think: str = None
     chat_messages: list = None
-    stats: dict = field(default_factory=dict)
-    markup_page: str = ""
+    stats: dict = None
```

Comment:
why did you put it to None? 
I just put it to dict (I think it was making something fail somewhere)